package.json

```diff
@@ -51,7 +51,7 @@
   "dependencies": {
     "ember-cli-babel": "^5.1.6",
     "chalk": "^1.1.3",
-    "ember-lodash": "^0.0.10",
```

Comment:
Version `^0.0.10` will clash with other addons defining the same dependency with a current release, e.g. `^4.0`. The thing about `dependencies` in ember addons is that the last one seems to win. In my case  ember-cli-raygun's dependency for ember-lodash beat out ember-cli-mirage's dependency.